getting-started/quick-start.md

```diff
@@ -0,0 +1,60 @@
+# Quick Start
+
+Axon provides a _**Quick Start Toolkit**_ to familarize yourself with the setup required for Axon Framework and Axon Server SE \(Standard Edition\). 
+
+The only pre-requisite to run the Quick Start is to have a Java 8+ JRE in your system.
+
+### Quick Start Toolkit Download
+
+The Quick Start Toolkit package is available for download at the following location -&gt; [https://axoniq.io/download](https://axoniq.io/download)
+
+This package contains,
+
+* Axon Framework Binaries
+* Axon Server Standard Edition
```

Comment:
In this section, there is _nothing_ about AxonServer. How about adding a list of relevant blogs in the resources section.

Reply:
Added Axon Server blogs